md/bpm-api.md

```diff
@@ -481,6 +481,18 @@ Retrieve humanTask objects that match the specified filters.
   `GET`
 * **Data Params**  
   [Standard search parameters](rest-api-overview.md#resource_search) are available.  
+  
+  * o: only parameters available in HumanTaskInstanceSearchDescriptor.java and in the result are accepted: 
```

Comment:
I agree with you, but the point is "how can we ensure to have a correct documentation" ? because the truth in this story said Anthony is "what you have in the JAVA CLASS."
So, maybe someone can add a comment in the JAVA class to say "attention, if you modify this class, please update the REST API doc ?"
Because you don't have time to check everywhere, it's more simple to say in the documentation "the real pointer is HERE". Remember here we have a Case in support because the doc is wrong, and we apologize to the customer for that. Idea is "what is the best to NOT HAVE THIS a second time ?" This is one idea, ideas are welcome.